pkg/appmanager/appManager_test.go

```diff
@@ -2464,6 +2464,15 @@ var _ = Describe("AppManager Tests", func() {
 				Expect(r).To(BeTrue(), "Ingress resource should be processed.")
 				Expect(resources.Count()).To(Equal(0))
 
+				// Shouldn't process Ingress with non-F5 class
+				ingressNotf5 := test.NewIngress("ingress-bad", "1", namespace, ingressConfig,
```

Comment:
Could you add a comment here pointing back to the bug?

// https://github.com/F5Networks/k8s-bigip-ctlr/issues/311